pom.xml

```diff
@@ -49,6 +49,8 @@
     <osgi.bundles.version>2.5.0</osgi.bundles.version>
 
     <spring.version>3.2.17.RELEASE</spring.version>
+    
```

Comment:
Do we really need these properties? Isn't `project.parent.basedir` the basedir of the sonatype parent? Can't we just use `../foobar` to get the basedir in the modules?

Reply:
For one level structure, we wouldn't need that property, but because we have a two level structure for spring boot modules, we need that property, otherwise we would have to reconfigure `org.eclipse.virgo.bundlor.maven` for the 2nd-level modules.

In other words, for 1st-level modules we would need to have:

```xml
<osgiProfilePath>../java6-server.profile</osgiProfilePath>
```
and for 2nd-level modules it would be:


```xml
<osgiProfilePath>../../java6-server.profile</osgiProfilePath>
```